### PR TITLE
Upload bugs

### PR DESCRIPTION
**Pull request by Kagiri2**
Fixed sidebar upload bug, added some styling to the posts and made the navbar smoother for mobile

### CODE REVIEW

File: src/web/src/Components/Posts.tsx
Comment (shimupan) [inline src/web/src/Components/Posts.tsx:1]: The layout of this file is kinda messed up, like the spacing

**Review by shimupan — CHANGES_REQUESTED** [review]
Just the spacing for Post.tsx is off. Also I think we should include pictures for issues and pull requests in the future. Could you include some before and after pictures here?

File: src/web/src/Components/Posts.tsx
Comment (Kagiri2) [reply src/web/src/Components/Posts.tsx:1]: yo turn on the backend

Comment (Kagiri2) [reply src/web/src/Components/Posts.tsx:1]: nvm

**Comment by Kagiri2** [comment]
Before:
Screenshot 2023-12-23 144922
After:
Screenshot 2023-12-23 144935

**Review by shimupan — APPROVED** [review]

**Pull request merged**